frontend/src/metabase/reference/metrics/MetricQuestions.jsx

```diff
@@ -49,75 +43,73 @@ const mapStateToProps = (state, props) => ({
   metric: getMetric(state, props),
   table: getTable(state, props),
   metadata: getMetadata(state),
-  entities: getMetricQuestions(state, props),
-  loading: getLoading(state, props),
-  loadingError: getError(state, props),
 });
 
 const mapDispatchToProps = {
   ...metadataActions,
 };
 
-class MetricQuestions extends Component {
-  static propTypes = {
-    style: PropTypes.object.isRequired,
-    entities: PropTypes.object.isRequired,
-    loading: PropTypes.bool,
-    loadingError: PropTypes.object,
-    metric: PropTypes.object,
-    table: PropTypes.object,
-    metadata: PropTypes.object.isRequired,
-  };
+const getDescription = question => {
+  const timestamp = moment(question.getCreatedAt()).fromNow();
+  const author = question.getCreator().common_name;
+  return t`Created ${timestamp} by ${author}`;
+};
 
-  render() {
-    const { entities, style, loadingError, loading, table, metric, metadata } =
-      this.props;
+export const MetricQuestions = ({ style, table, metric, metadata }) => {
+  const {
+    data = [],
+    isLoading,
+    error,
+  } = useQuestionListQuery({
+    query: { f: "using_metric", model_id: metric.id },
+    enabled: true,
```

Comment:
```suggestion
```

Reply:
Thank you
I'll remove it both here and in SegmentQuestions when I rebase and before I merge https://github.com/metabase/metabase/pull/38500